unit_tests/testHDF5FileType/testHDF5FileType.f90

```diff
@@ -249,14 +249,11 @@ SUBROUTINE testHDF5FileTypeSetup()
   !write the reference file for the read side
   CALL h5%init('readtest.h5','NEW')
   CALL h5%fopen()
-  CALL h5%mkdir('groupB')
   CALL h5%fwrite('groupB->memB0',refB0)
   CALL h5%fwrite('groupB->memB1',refB1)
   CALL h5%fwrite('groupB->memB2',refB2)
   CALL h5%fwrite('groupB->memB3',refB3)
   CALL h5%mkdir('groupC')
-  CALL h5%mkdir('groupC->anotherGroup')
-  CALL h5%mkdir('groupC->anotherGroup->moreGroups')
```

Comment:
the best unit testing is the unit testing that removes unit testing lines will increasing unit testing coverage